modules/odo-getting-started-code-nodejs.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-nodejs.adoc
+
+:_content-type: Procedure
+[id="odo-getting-started-nodejs_{context}"]
+
+= Generating sample code for Node.js 
+
+Use the https://expressjs.com/[Express] framework to generate the sample code.
+
+.Prerequisites
+* You have installed Node.js and supporting tools
+
+.Procedure 
+
+. Install Express:
++
+[source,terminal]
+----
+npm install express --save
+----
+
+
+. Generate an example project:
++
+.Sample command
+[source, terminal]
```

Comment:
```suggestion
[source,terminal]
```

Reply:
I don't believe whitespace is significant here